tests/resources/test-types.json

```diff
@@ -3790,13 +4094,7 @@
         ],
         "forVehicleSize": null,
         "forVehicleConfiguration": null,
-        "forVehicleAxles": [
```

Comment:
Don't see an explanation for this on the ticket.

Reply:
This basically means that this test will be displayed for any number of axles. I wrote this one differently because it's the only test type that includes vehicles with any number of axles (1-10). 
The filtering based on vehicle axle numbers is done on the FE